.github/workflows/pr-operator.yml

```diff
@@ -0,0 +1,400 @@
+name: pr-operator
+on:
+  workflow_call:
+    inputs:
+      OPERATOR_SDK_VERSION:
+        description: "Version of Operator SDK Binary for installation"
+        default: "v1.9.0"
+        required: false
+        type: string
+      BUILD_PLATFORMS:
+        description: 'Comma separated list of targets for builds e.g. "linux/amd64,linux/arm64,linux/ppc64le"'
+        default: "linux/amd64,linux/arm64,linux/ppc64le"
```

Comment:
please add z/OS here in the default. The issue that was preventing us from building zos images should have been resolved.

Reply:
Addressed